assets/queries/ansible/aws/iam_policy_grants_full_permissions/query.rego

```diff
@@ -15,6 +15,7 @@ CxPolicy[result] {
 
 	common_lib.is_allow_effect(statement)
 	common_lib.equalsOrInArray(statement.Resource, "*")
+	common_lib.not_only_ec2messages_actions(statement.Action)
```

Comment:
I think we should look to the CF query approach and implement it in ANS and TF query